sweeper/sweeper.go

```diff
@@ -83,7 +83,7 @@ func (s *Sweeper) threadedPerformSweep() {
 	// Kick off a skyd client cache rebuild. That happens in a separate
 	// goroutine. We'll block on the result channel only after we're done with
 	// the other tasks we can do while waiting.
-	res := s.staticSkydClient.RebuildCache()
+	res := s.staticSkydClient.RebuildCache(true)
```

Comment:
If we always call it with `true` we might not need the argument?

Reply:
I initially started without the argument. But then I thought that it's quite limiting to always exclude those blocked skylinks. Maybe we'll need the cache for something else as well like listing all the files pinned by `skyd`? Maybe it's a case of "you ain't gonna need it". I'll remove the argument - it's easy to add later if we need it.